docs/javascript/v2/features/notifications.mdx

```diff
@@ -207,6 +207,74 @@ function Notification() {
 
 </Tab>
 
+## Listening to specific notifications
+
+You can pass a single notification or array of notifications from one of `HMSNotificationTypes`.
+
+<Tabs id="notification-types" items={['JavaScript', 'React']} />
+
+<Tab id="notification-types-0">
+
+```jsx{14-44}
+import { hmsNotifications } from './hms';
+
+const unsubscribe = hmsNotifications.onNotification((notification) => {
+    if (!notification) {
+        return;
+    }
+
+    console.log('notification type', notification.type);
+    console.log('data', notification.data);
+}, HMSNotificationTypes.PEER_JOINED);
+
+const unsubscribePeerNotifications = hmsNotifications.onNotification((notification) => {
+    if (!notification) {
+        return;
+    }
+    // this will be called when one of HMSNotificationTypes.PEER_JOINED, HMSNotificationTypes.PEER_LEFT is arrived
+    console.log('notification type', notification.type);
+    console.log('data', notification.data);
+}, [HMSNotificationTypes.PEER_JOINED, HMSNotificationTypes.PEER_LEFT]);
+```
+
+</Tab>
+
+<Tab id="notification-types-1">
+
+```jsx{14-44}
+import { useHMSNotifications } from '@100mslive/react-sdk';
+
+function Notification() {
+    const notification = useHMSNotifications(HMSNotificationTypes.PEER_JOINED);
+    const peerNotification = useHMSNotifications(HMSNotificationTypes.PEER_JOINED);
+
+
+    React.useEffect(() => {
```

Comment:
why two use effects?

Reply:
to keep them separate. handling will be slightly different.

Reply:
easy to copy paste depending on what is used.

Reply:
what's the difference between them both are `useHMSNotifications(HMSNotificationTypes.PEER_JOINED)`, right?